src/mrt/project.rs

```diff
@@ -0,0 +1,43 @@
+use std::path::PathBuf;
+use glob::glob;
+
+use crate::package::Package;
+use crate::manifest::Manifest;
+
+pub struct Project {
+    path: PathBuf,
+    manifest: Manifest,
+}
+
+impl Project {
+    pub fn new(path: PathBuf) -> Project {
+        Project {
+            path,
+            manifest: Manifest::new()
+        }
+    }
+
+    pub fn get_packages(&self) -> Vec<Package> {
+
+        let mut packages: Vec<Package> = vec![];
+        
+        for package_glob in &self.manifest.packages {
+            let full_glob = self.path.parent().unwrap()
+                .join(&package_glob);
+        
+            for entry in glob(&full_glob.to_str().unwrap())
+                    .expect("Failed to read glob pattern") {
```

Comment:
Most likely for debugging you would like to know what glob failed.

Reply:
True. Addressed. Also removed all unwraps from this file. Will remove unwraps for other files too.